Dockerfile

```diff
@@ -18,6 +18,13 @@ RUN \
 RUN groupadd -g 10001 app && \
     useradd -M -u 10001 -g 10001 -G app -d /app -s /sbin/nologin app
 
+RUN \
+    mkdir /app/cache && \
+    chgrp app /app/cache && \
+    chmod g+w /app/cache
```

Comment:
u+w

Reply:
I did `chgrp`
I can change to `chown`

Reply:
We should probably do it the other way and use `$CACHE_FOLDER` :)